.github/workflows/builder_gradle_slsa3.yml

```diff
@@ -40,7 +40,7 @@ on:
       provenance-name:
         description: "The file name of the attestation upload artifact."
         # NOTE: This is an "untrusted" value returned from the build.
-        value: "${{ fromJSON(jobs.slsa-run.outputs.build-artifacts-outputs).attestation-name }}"
```

Comment:
I'm not sure if we should update these. They are only used by BYOB integrations. If we want to update them, we'd also need to update our internal usage of it, like https://github.com/slsa-framework/slsa-github-generator/blob/main/internal/builders/gradle/create_attestation.sh, same for maven, etc.
I'm thinking we can skip them and keep this PR simpler

Let's keep it simple and only update the option for the generic generator.